File: python/python_fundamentals/functions/functions_exercise/odd_and_even_sum.py
def sum(the_number):
    result_of_odd = 0
    result_of_even = 0

    for number in the_number:
        integer_number = int(number)

        if integer_number % 2 == 0:
            result_of_even += integer_number
        elif integer_number % 2 != 0:
            result_of_odd += integer_number


    return f"Odd sum = {result_of_odd}, Even sum = {result_of_even}"

some_number = input()
print(sum(some_number))

File: Front-End JS/JS Front-End/FunctionsAndStatementsExercises/OddAndEvenSum.js
function oddEven(number) {
    let sumOdd = 0
    let sumEven = 0
    let numberString = String(number)

    for (let i = 0; i < numberString.length; i++) {
        numberNum = Number(numberString[i])

        if (numberNum % 2 == 0){
            sumEven += numberNum
        }
        else {
            sumOdd += numberNum
        }
    }

    return `Odd sum = ${sumOdd}, Even sum = ${sumEven}`
    
}

console.log(oddEven(1000435))